h/static/scripts/directive/annotation.js

```diff
@@ -707,7 +707,10 @@ function AnnotationController(
 
   vm.share = function(event) {
     var $container = angular.element(event.currentTarget).parent();
-    $container.addClass('open').find('input').focus().select();
+    $container.addClass('open');
```

Comment:
The changes to the directives here are about compatibility with `jqLite` instead of the full version of `jQuery`